index1.html

```diff
@@ -0,0 +1,143 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ <meta charset="UTF-8">
+ <meta http-equiv="X-UA-Compatible" content="IE=edge">
+ <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.2/css/all.css" integrity="sha384-*********************" crossorigin="anonymous"> 
+        <link rel="stylesheet" href="style1.css">
+        <link href='http://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+ <title>Zoum-Capstone-project</title>
+</head>
+<body class="body">
+<div class="firstFace">  </div>    
+
+ <!-- <header>
+       <div class="toggle">
+              <i class="fas fa-bars ouvrir"></i> 
+              <i class="fas fa-times fermer"></i>         
+          </div>
+          <div class="menu">
+              <li class="content-liA"><a href="index1.html">About</a></li>
+              <li class="content-liA"><a href="#">program</a></li>
+              <li class="content-liA"><a href="#">Join</a></li>
+              <li class="content-liA"><a href="#">Sponsor</a></li>
+              <li class="content-liP"><a href="#">News</a></li>
+              <li class="content-liC"><a href="#">CC compaign</a></li>
+         
+          </div>
+
+          <div class="header-desk">
+              <div class="header-desk">
+                     <li><a href="#" target="_blank"><i class="fab fa-facebook"></i></a></li>
+                     <li> <a href="https://twitter.com/zoumaniguimass1" target="_blank" rel="noopener noreferrer"><i class="fab fa-twitter"></i></a></li>
+                     <ul class=" nav-headerfirst">
+                            <li><a href="#">English</a></li>
+                            <li><a href="#">My page</a></li>
+                            <li><a href="#">loyout</a></li>
+                     </ul>
+              </div>
+              <div class="header-second">
+               <div class="logo"><a href="index.html"><img src="images/icon.png " width="100%" height="80%" alt=""></a></div>
+               <nav>
+                      <ul>
+                             <li class="content-liA"><a href="index1.html">About</a></li>
+                             <li class="content-liA"><a href="#">program</a></li>
+                             <li class="content-liA"><a href="#">Join</a></li>
+                             <li class="content-liA"><a href="#">Sponsor</a></li>
+                             <li class="content-liP"><a href="#">News</a></li>
+                             <li class="content-liCc"><a href="#">CC compaign</a></li>
+                      </ul>
+               </nav>
+        </div>
+              </div>
+          </div>
+ </header> -->
```

Comment:
[OPTIONAL] Kindly remove unused code from your code base, it takes up space, causes confusion, and leads to maintenance issues if it's not removed. 🥺